lust/lib/models/capacity/trend.dart

```diff
@@ -0,0 +1,6 @@
+/// Enum to describe the trend of the filling status of the library
+
+enum Trend {
+  /// !!! Order determines the display of the trend in capacity
+  rising, leveling, falling
+}
```

Comment:
Please add an empty line at the end of each file.

Reply:
done